gdbgui/src/js/GdbApi.jsx

```diff
@@ -249,13 +249,21 @@ const GdbApi = {
       );
     }, WAIT_TIME_SEC * 1000);
   },
+  /**
+   * Set if the commands must be sent on all user sessions
+   * @param state mpi state
+   * @return nothing
+   */
+  set_mpi_state: function(state) {
```

Comment:
All files are now typescript, so arguments like `state` can be annotate with types now

Reply:
Yeah I saw merging with master ... I am already changing all of them doing it.